app/views/helpers/logs_pagination.phtml

```diff
@@ -6,46 +6,51 @@
 ?>
 
 <?php if ($this->nbPage > 1) { ?>
+	<nav class="nav-pagination nav-list">
 <ul class="pagination">
 	<?php
 		/** @var int $getteur */
 		$params[$getteur] = 1;
 	?>
 	<li class="item pager-first">
-		<?php if ($this->currentPage > 1) { ?>
 		<a href="<?= Minz_Url::display(array('c' => $c, 'a' => $a, 'params' => $params)) ?>">« <?= _t('gen.pagination.first') ?></a>
-		<?php } ?>
 	</li>
 
-	<?php $params[$getteur] = $this->currentPage - 1; ?>
-	<li class="item pager-previous">
-		<?php if ($this->currentPage > 1) { ?>
-		<a href="<?= Minz_Url::display(array('c' => $c, 'a' => $a, 'params' => $params)) ?>">‹ <?= _t('gen.pagination.previous') ?></a>
+		<?php $params[$getteur] = $this->currentPage - 1; ?>
+		<li class="item pager-previous">
+			<a href="<?= Minz_Url::display(array('c' => $c, 'a' => $a, 'params' => $params)) ?>">‹ <?= _t('gen.pagination.previous') ?></a>
+		</li>
+
+		<?php if ($this->currentPage - 2 > 1) { ?>
+		<li class="item">...</a></li>
 		<?php } ?>
-	</li>
 
-	<?php for ($i = $this->currentPage - 2; $i <= $this->currentPage + 2; $i++) { ?>
-		<?php if($i > 0 && $i <= $this->nbPage) { ?>
-			<?php if ($i != $this->currentPage) { ?>
-			<?php $params[$getteur] = $i; ?>
-			<li class="item pager-item"><a href="<?= Minz_Url::display(array('c' => $c, 'a' => $a, 'params' => $params)) ?>"><?= $i ?></a></li>
-			<?php } else { ?>
-			<li class="item pager-current"><?= $i ?></li>
+		<?php for ($i = $this->currentPage - 2; $i <= $this->currentPage + 2; $i++) { ?>
+			<?php if($i > 0 && $i <= $this->nbPage) { ?>
+				<?php if ($i != $this->currentPage) { 
+					$params[$getteur] = $i; 
+					$class = '';
+					$aria = 'false';
+				} else { 
+					$class = ' active';
```

Comment:
Presumably that should be either .= or 'active'?

Reply:
it is correct.
The white space is used to separate it from "item" class. If `$class = ''`, than there should not be an white space:
`<li class="item<?= $class ?>">`